main.js

```diff
@@ -0,0 +1,160 @@
+// let player = playerX;
+// let winner = null;
+let turnCount = 1;
+let whosTurn = document.getElementById("whosTurn");
+
+let cell_0 = document.getElementById("cell-0");
+let cell_1 = document.getElementById("cell-1");
+let cell_2 = document.getElementById("cell-2");
+let cell_3 = document.getElementById("cell-3");
+let cell_4 = document.getElementById("cell-4");
+let cell_5 = document.getElementById("cell-5");
+let cell_6 = document.getElementById("cell-6");
+let cell_7 = document.getElementById("cell-7");
+let cell_8 = document.getElementById("cell-8");
+
+// * Play function initiates only after Let's Go button has been activated and subsequently disabled.
+// * Alert message triggers if claimed cell is already populated with "X" or "O"
+
+function play(e) {
+  let move = turnCount % 2 ? "X" : "O";
+  let nextMove = turnCount % 2 ? "O" : "X";
+  if (e.target.textContent === "") {
+    turnCount++;
+    e.target.textContent = move;
+    if (
```

Comment:
The brute force approach certainly works, but you could remove half this code by checking against `move` rather than the literal string X or O since you've already got that variable in here. You could condense this further by using some sort of iterator such as a for of loop to check against all possible win conditions.